pkg/controller/history/controller_history.go

```diff
@@ -454,3 +452,31 @@ func (fh *fakeHistory) ReleaseControllerRevision(parent metav1.Object, revision
 	}
 	return clone, fh.indexer.Update(clone)
 }
+
+func getPodLabelsFromData(data runtime.RawExtension) map[string]string {
+	var dat map[string]interface{}
+	labelMap := make(map[string]string)
+	if err := json.Unmarshal(data.Raw, &dat); err != nil {
```

Comment:
How do we know that the raw extension will always be json and not say protobuf?

Reply:
@cheftako Please have a look at my recent commit :)